app/views/partials/_navbar.html.haml

```diff
@@ -0,0 +1,14 @@
+%nav#navbar.navbar.navbar-light{:style => "background-color: black;"}
+  %ul.nav.justify-content-start{:style => "padding-left: 1em;"}
```

Comment:
Please avoid setting style attributes. It is always better to create new css classes or expand existing ones.